### PR TITLE
OSDOCS-2461: Created a new book covering the OpenShift Cluster Manager.

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR creates a new book covering the OCM tool.

JIRA: https://issues.redhat.com/browse/OSDOCS-2461

Preview: https://deploy-preview-38655--osdocs.netlify.app/openshift-dedicated/latest/ocm/ocm-overview

### CODE REVIEW

**Comment by EricPonvelle** [comment]
@okashi18 PTAL at this first draft of the OCM topic. I had a few items I had to take a guess on.

**Comment by EricPonvelle** [comment]
@jharrington22 and @cblecker - can you take a look at this PR and let me know if I covered everything? Also, do either of you know the QE for this?

File: modules/ocm-accesscontrol-tab.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+.Prerequisites
+
+. You must be the cluster owner or have the correct permissions to grant roles on the cluster.
+
+.Procedure
+
+. Select the **Grant role** button.
+. Enter the Red Hat account login for the user that you wish to grant a role on the cluster.
+. Select the **Grant role** button on the dialog box.
+. [I couldn't test past this point as no names worked to grant access.]
```
Comment (cblecker) [inline modules/ocm-accesscontrol-tab.adoc:19]: L18 is this last step. If the username is correct, the dialog should close and will list the user has having "Cluster Editor" access.

**Review by cblecker — COMMENTED** [review]
One comment, but seems fine to me.

@gregsheremeta Any comments from a PM perspective?

**Comment by EricPonvelle** [comment]
PR continued at https://github.com/openshift/openshift-docs/pull/39178

**Pull request closed without merging**